src/colors.js

```diff
@@ -1,7 +1,12 @@
 export const colors = {
+  mainGray: '#373737',
+  dialodsGray:
```

Comment:
Can we have a dialod about it @Batsvinova-Anastasiia?)

Reply:
Done ✅